install_config/aggregate_logging.adoc

```diff
@@ -355,7 +344,7 @@ $ oc logs -f <pod_name>
 ----
 
 ifdef::openshift-enterprise[]
-. As a cluster administrator, deploy a template that is created by the deployer:
+. As a cluster administrator, deploy the template that the deployer created:
```

Comment:
well, it creates several and this is just one of them, but if you think this is clearer...